pyproject.toml

```diff
@@ -44,7 +45,6 @@ pytest-cov = "^4.0.0"
 docs = ["sphinx"]
 
 [tool.poetry.build]
-script = "build.py"
 generate-setup-file = true
 
 [build-system]
```

Comment:
I can't leave a suggestion but you can now remove:

- `setuptools`
- `cython`

Reply:
Fixed